yahoo-finance.gemspec

```diff
@@ -1,16 +1,17 @@
-$LOAD_PATH << '.'
-require 'lib/yahoo_finance'
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'yahoo-finance/version'
 
 spec = Gem::Specification.new do |s|
-  s.name = "yahoo-finance"
+  s.name = 'yahoo-finance'
```

Comment:
I agree. I just use a ruby linter and it tells me to use single quotes when string interpolation is not needed. So I am used to that.